submissions/idiagheOsaigbovo/CssTask/Registration form/Regform.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>AfriDirect</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+
+    <h1>AfriDirect Registration Form</h1>
+
+
+    <div class="registration">
+               
+        <form id="registration" method="post" action="..php"></form>
+        
+        <h1>Registration</h1>
+        
+        <input type="text" name="Fname" id="name" placeholder="Enter firstname"><br>
```

Comment:
@osayiosas ids are meant to be unique something that identifies the tag and its function uniquely. You will have to change the other tags with id "name". Also, "name" is an attribute, use another value say "fname" or "firstname"

Reply:
you can check this out now